pants.ci.toml

```diff
@@ -1,30 +1,9 @@
 [GLOBAL]
 colors = true
 
-remote_cache_read = true
-remote_cache_write = true
```

Comment:
This is going to hurt, isn't it?

I think this needs to be shouted over in #development at least...